mlrun/api/db/base.py

```diff
@@ -67,7 +67,7 @@ def list_runs(
         self,
         session,
         name="",
-        uid=None,
+        uid: Union[str, List[str]] = None,
```

Comment:
```suggestion
        uid: Union[str, List[str],None] = None,
```

Reply:
In general, I prefer the approach of using `Optional` instead, since it does refer that logically the parameter is optional, and that's why it will be `None`. It is equivalent to adding `None` in the `Union` but I find it better. So, that's what I did :-)